Library/Homebrew/livecheck/strategy/gnome.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module Homebrew
+  module Livecheck
+    module Strategy
+      module Gnome
+        module_function
+
+        NICE_NAME = "GNOME"
+
+        # Formulae that do not use GNOME's "even-numbered minor is stable" scheme
+        # "libart_lgpl" is the package name for libart
+        DEV_VERSION_ALLOWLIST = %w[
+          gcab
+          gtk-doc
+          gtk-mac-integration
+          libart_lgpl
+          libepoxy
+        ].freeze
+
+        def match?(url)
+          /download\.gnome\.org/.match?(url)
+        end
+
+        def find_versions(url, regex = nil)
+          package_name = url.match(%r{/sources/(.*?)/})[1]
+
+          page_url = "https://download.gnome.org/sources/#{package_name}/cache.json"
+          regex ||= if DEV_VERSION_ALLOWLIST.include?(package_name)
+            /#{Regexp.escape(package_name)}-(\d+(?:\.\d+)+)\.t/
+          else
+            # Only match versions with an even-numbered minor (except x.90+)
```

Comment:
This is a good comment. I'd like to see more comments like this in other strategies.